app/policies/user_policy.rb

```diff
@@ -0,0 +1,28 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
+    true
+  end
+
+  def feed?
+    true
```

Comment:
Might want to change this so that only the current_user can see their feed and discover, and not other users'.